components/Button.js

```diff
@@ -57,7 +57,7 @@ const LinkButton = styled(Link)`
   font-size: var(--fs-caption);
 
   &:hover {
-    background-color: rgba(45, 57, 151, 1);
+    background-color: rgba(233, 150, 209, 1);
     font-weight: 600;
```

Comment:
Instead of defining multiple buttons you can think about using button variants. This can reduce code and make it less repetitive.